packages/uniforms/src/createSchemaBridge.js

```diff
@@ -1,13 +1,15 @@
 import invariant from 'fbjs/lib/invariant';
 
 import GraphQLBridge       from './GraphQLBridge';
-import SimpleSchemaBridge  from './SimpleSchemaBridge';
+import JSONSchemaBridge    from './JSONSchemaBridge';
 import SimpleSchema2Bridge from './SimpleSchema2Bridge';
+import SimpleSchemaBridge  from './SimpleSchemaBridge';
 
 const bridges = [
     GraphQLBridge,
     SimpleSchemaBridge,
-    SimpleSchema2Bridge
+    SimpleSchema2Bridge,
+    JSONSchemaBridge // Other schemas are often valid JSON schema, so we check this at the end
```

Comment:
_(as we discussed it internally)_

It shouldn't be order-dependent because of splitting bridges later.